go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/fraunhoferfokus/deckschrubber
```

Comment:
PR #29 also addresses go modules, however, I haven't come round to have it merge. It would be great if we merge that one first and built this PR on the top. Otherwise this PR would contain too many changes at once.

Reply:
Will wait for this to merge first